### ISSUE TITLE
Home page

### ISSUE THREAD

**Issue by rasmus-storjohann-PG** [pg-irc/pathways-frontend#107]
Home screen with sections representing the different parts of the app functionality:

![screenshot from 2018-06-08 14-09-07](https://user-images.githubusercontent.com/32307781/41181140-91cb27a6-6b25-11e8-9c7a-28ce33590ca1.png)

* Learn about (blocked by #108 for explore all component)
* Plan for success (blocked by #82 for task list component)
* Out of scope: Dynamic feature card
* In scope: include copyright notice on home page, see #79 for text
